react-ui/src/env.js

```diff
@@ -0,0 +1,7 @@
+// WARNING! All process.env values are strings, so check for 'true' or 'false'
+
+const {
+  NODE_ENV,
```

Comment:
Does react-ui have access to NODE_ENV? Where is this being used?

Reply:
This is used in [serviceWorker.js](https://github.com/heshammourad/vexillology-contests/blob/master/react-ui/src/serviceWorker.js). Code was already using `process.env.NODE_ENV === 'production'` in the initial commit, so I assume react-ui has access?